src/components/profile/ConnectedAccounts.js

```diff
@@ -0,0 +1,196 @@
+// @flow
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import { Switch, Text, View } from 'react-native'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { CeramicSDK } from '@gooddollar/ceramic-seed-sdk'
+import Web3 from 'web3'
+import { withStyles } from '../../lib/styles'
+import { Section, Wrapper } from '../common'
+
+import useTorus from '../../components/auth/torus/hooks/useTorus'
+import GoogleBtnIcon from '../../assets/Auth/btn_google.svg'
+import FacebookBtnIcon from '../../assets/Auth/btn_facebook.svg'
+import MobileBtnIcon from '../../assets/Auth/btn_mobile.svg'
+import Recaptcha from '../../components/auth/login/Recaptcha/index'
+import { useErrorDialog } from '../../lib/undux/utils/dialog'
+
+const TITLE = 'Connected Accounts'
+const sdk = new CeramicSDK('https://ceramic-clay.3boxlabs.com')
+
+const ConnectedAccounts = ({ screenProps, styles }) => {
+  const [providers, setProviders] = useState([])
+  const [authenticatorData, setAuthenticatorData] = useState({})
+
+  const initializeAccount = useCallback(async () => {
+    const privateAccountKey = JSON.parse(localStorage.getItem('GD_masterSeed'))
+    const web3 = new Web3()
+    const publicAccountKey = web3.eth.accounts.privateKeyToAccount(privateAccountKey)
+    await sdk.initialize(privateAccountKey.slice(0, privateAccountKey.length / 2), publicAccountKey.address)
+  }, [])
```

Comment:
this should be a memo or effect, you want to run that only once.
you can make it as an hook useCeramicSDK

Reply:
made a hook